flocker/common/script.py

```diff
@@ -22,6 +22,12 @@
 
 from .. import __version__
 
+try:
+    from ._journald import sd_journal_send
+except OSError:
+    # This platform doens't have journald.
+    pass
```

Comment:
This will make `--journald` behavior on non-journald-having platforms pretty unfriendly.  Rejecting `--journald` in this case with a message would be nice.